test_scripts/Defects/8_1/3804_1_RAI_use_hmiCapabilities_from_cache_file.lua

```diff
@@ -0,0 +1,103 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/smartdevicelink/sdl_core/issues/3802
+---------------------------------------------------------------------------------------------------
+-- Description: Check that SDL sends RegisterAppInterface response with the <hmiCapabilities> parameter
+--  from HMI capabilities cache file (hmi_capabilities_cache.json) in case appropriate <systemCapabilities>
+--  parameter is missing
+--
+-- Preconditions:
+-- 1. HMI capabilities cache file (hmi_capabilities_cache.json) exists:
+--  - with "hmiCapabilities": <hmiCapabilities> parameter
+--  - without the "systemCapabilities":<systemCapabilities> parameter
+-- 2. SDL, HMI, Mobile session are started
+-- Steps:
+-- 1. App sends RegisterAppInterface requests
+-- SDL does:
+--  - send RegisterAppInterface response with the <hmiCapabilities> parameter from HMI capabilities cache file
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('user_modules/sequences/actions')
+local utils = require("user_modules/utils")
+local color = require("user_modules/consts").color
+local SDL = require('SDL')
+local hmi_values = require('user_modules/hmi_values')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local missingCapability = {
+  [1] = "navigationCapability",
+  [2] = "phoneCapability",
+  [3] = "videoStreamingCapability"
+}
+
+--[[ Local Functions ]]
+local function ignitionOff()
+  local isOnSDLCloseSent = false
+  common.getHMIConnection():SendNotification("BasicCommunication.OnExitAllApplications", { reason = "SUSPEND" })
```

Comment:
Using two styles of call (sendNotification / ExpectNotification and etc) in a file:
1. in line 39 and other - common.**getHMIConnection**():SendNotification 
2. in line 72 - common.**hmi.getConnection()**:ExpectNotification 

Could you lead to one of style? (Better use style №2)